packer/create-ami.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
Add the copyright notice here.  Look at the top of the CFn template to copy/paste.